fdbcli/Util.actor.cpp

```diff
@@ -153,4 +153,31 @@ ACTOR Future<bool> getWorkers(Reference<IDatabase> db, std::vector<ProcessData>*
 	}
 }
 
+ACTOR Future<Void> getStorageServerInterfaces(Reference<IDatabase> db,
+                                              std::map<std::string, StorageServerInterface>* interfaces) {
+	state Reference<ITransaction> tr = db->createTransaction();
+	state int i = 0;
```

Comment:
why is this a state variable?

Reply:
not need to be.
Copied from another function where there's a wait inside for-loop.